src/lib/journey/callbacks/suspend-text-output/suspend-text-output.svelte

```diff
@@ -0,0 +1,11 @@
+<script lang="ts">
```

Comment:
There should already be a directory named `suspended-text-output`, can you use that directory and the same word, "suspended", for the filename. That way, it will match the callback name.